### PR TITLE
blog(embertimes164): Async Data and Autotracking

### PR DESCRIPTION
**Pull request by chrisrng**
Covered Chris K's blog post

### CODE REVIEW

File: source/2020-09-25-the-ember-times-issue-164.md [hunk 1]
```diff
@@ -88,15 +88,36 @@ The blog also discusses using [`keyForAttribute`](https://api.emberjs.com/ember-
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
+## [Blog Post: Async Data and Autotracking in Ember Octane ✨](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/)
 
-## [Section title in sentence case 🐹](section-url)
+A new blog post from [Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) talks about [Async Data and Autotracking in Ember Octane](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/). This blog post continues off his previous blog on [Migrating Off of PromiseProxyMixin in Ember Octane](https://v5.chriskrycho.com/journal/migrating-off-of-promiseproxymixin-in-ember-octane/) where Chris describes a way to divest out of the mixin/inheritance approach and into a composition-based approach. He does this by using a [load helper](https://gist.github.com/chriskrycho/306a82990dd82203073272e055df5cd1) and a new `AsyncData` structure.
```
Comment (ijlee2) [inline source/2020-09-25-the-ember-times-issue-164.md]: I simplified the last 2 sentences in the suggestion. What do you think? (Cool word--divest. Don't think I've heard of it before.)

```suggestion
A new blog post from [Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) talks about [Async Data and Autotracking in Ember Octane](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/). This blog post is a continuation of [Migrating Off of PromiseProxyMixin in Ember Octane](https://v5.chriskrycho.com/journal/migrating-off-of-promiseproxymixin-in-ember-octane/), which describes a way to divest mixin and inheritance in favor of composition. Chris does this by using a [load helper](https://gist.github.com/chriskrycho/306a82990dd82203073272e055df5cd1) and a new `AsyncData` structure.
```

**Review by ijlee2 — APPROVED** [review]
Thanks for reading Chris' blog post and highlighting the key points. I haven't read it yet and look forward to soon!

File: source/2020-09-25-the-ember-times-issue-164.md [hunk 2]
```diff
@@ -88,15 +88,36 @@ The blog also discusses using [`keyForAttribute`](https://api.emberjs.com/ember-
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+The new blog goes through treating that `AsyncData` as just data but at the same time making sure to handle all of its states (loading, loaded, error). By integrating with tracked properties, the `AsyncData` class just reacts to its modeled states and internals, handling this transition for us. This enables us to access the data returned from the load helper as normal data and react based on its state.
```
Comment (ijlee2) [inline source/2020-09-25-the-ember-times-issue-164.md]: I noticed that the alex linter marked the word "just." What do you think about:

```suggestion
The new blog goes through treating that `AsyncData` as ordinary data, but making sure to handle all states (loading, loaded, error). By integrating with tracked properties, the `AsyncData` class reacts to its modeled states and internals, handling this transition for us. This enables us to access the data returned from the load helper as normal data and react based on its state.
```

File: source/2020-09-25-the-ember-times-issue-164.md [hunk 3]
```diff
@@ -88,15 +88,36 @@ The blog also discusses using [`keyForAttribute`](https://api.emberjs.com/ember-
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
+## [Blog Post: Async Data and Autotracking in Ember Octane ✨](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/)
```
Comment (amyrlam) [inline source/2020-09-25-the-ember-times-issue-164.md:92]:
```suggestion

## [Blog Post: Async Data and Autotracking in Ember Octane ✨](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/)
```

adding a blank line about the `##` will fix markdownlint

//

@chrisrng thanks for the awesome writeup and @ijlee2 for the review! 💯 

i'm going to start a delayed writeup on chris's mixin post so there will be a lot of chris content here!

**Comment by amyrlam** [comment]
i'm going to merge this in so i can add the related post above!

**Pull request merged**